File: pyg_manager_web/src/main/webapp/js/service/goodsDescService.js
//服务层
app.service("goodsDescService", function ($http) {

	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../goodsDesc/findAll.do');
	}
    //分页查询所有
    this.findByPage = function (pageNum,pageSize) {
        return $http.get('../goodsDesc/findByPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize);
    };
    //根据条件分页查询
    this.findByCondition = function (pageNum, pageSize, conditionObj) {
        return $http.post("../goodsDesc/findByCondition.do?pageNum=" + pageNum + "&pageSize=" + pageSize, conditionObj);
    };
    //添加
    this.add = function (obj) {
        return $http.post('../goodsDesc/add.do', obj);
    };
    //根据id查找
    this.findOneById = function (id) {
        return $http.get("../goodsDesc/findOneById.do?id=" + id);
    };
    //修改
    this.update = function (obj) {
        return $http.post("../goodsDesc/update.do", obj);
    };
    //批量删除
    this.batchDelete = function (idsArr) {
        return $http.post("../goodsDesc/batchDelete.do", idsArr);
    };

});

File: pyg_seller_interface/src/main/java/com/songxin/pyg/seller/service/GoodsDescService.java
package com.songxin.pyg.seller.service;
import java.util.List;
import com.songxin.pyg.pojo.TbGoodsDesc;

import com.songxin.pyg.vo.PageResultVO;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface GoodsDescService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbGoodsDesc> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResultVO findByPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbGoodsDesc goodsDesc);
	
	
	/**
	 * 修改
	 */
	public void update(TbGoodsDesc goodsDesc);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbGoodsDesc findOneById(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void batchDelete(Long[] ids);

	/**
	 * 根据条件分页查询
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResultVO findByCondition(TbGoodsDesc goodsDesc, int pageNum, int pageSize);
	
}
